src/mail.sh

```diff
@@ -123,6 +124,7 @@ function mail_send()
 
   [[ -n "$opts" ]] && cmd+=" $opts"
   [[ -n "$private" ]] && cmd+=" $private"
+  [[ -n "$rfc" ]] && cmd+=" $rfc"
```

Comment:
Is `--rfc` part of the git-send-email options? I did not find anything like that. Tbh, I expected something like `--subject-prefix="RFC"`

Reply:
git-send-email uses git-format-patch under the hood to create the email files, and that contains the ``--rfc`` option.

https://git-scm.com/docs/git-format-patch#Documentation/git-format-patch.txt---rfc